spec/lcm/integration/user_provisioning_end_to_end_spec.rb

```diff
@@ -6,56 +6,81 @@
 require_relative 'support/s3_helper'
 require_relative 'brick_runner'
 
-def uniq_label_with_value(project)
+def uniq_label_with_value(project, value_count)
```

Comment:
info: Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction) (reek)